src/shared/components/case-editor/case-edit/case-edit.component.spec.ts

```diff
@@ -563,7 +563,7 @@ describe('CaseEditComponent', () => {
           expect(component.form.get('data').get(CASE_FIELD_2.id)).not.toBeNull();
```

Comment:
I'm confused as to what these two assertions are testing... I don't see anything in here that asserts that the complex fields were _cleared_. In fact, this looks like it's _almost_ testing that they're _not_ cleared. 🤷‍♂️

Reply:
The same applies to the assertions on lines 585 and 586 but I can't put a comment there. 😂

Reply:
@paul-graham - I agree it's a bit confusing. I'm not sure what the original author's intention was here (it wasn't me!), but perhaps they forgot to add _not_ in the test description, i.e. "and _not_ clear hidden complex form field".

Reply:
So you're saying that it's testing that the value _doesn't_ get cleared out? If so, we should really rename the test or it's contradictory. 😬

Reply:
Yes - definitely. I'll add the missing "not" to the descriptions here and on Lines 585 and 586.